File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/body/ride/UpdateRideBody.java
package com.lattis.ellipse.data.network.model.body.ride;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 3/30/17.
 */

public class UpdateRideBody {

    @SerializedName("trip_id")
    private int trip_id;
    @SerializedName("steps")
    private double[][] steps;

    public UpdateRideBody(int trip_id,
                          double[][]steps) {
        this.trip_id = trip_id;
        this.steps=steps;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/ride/UpdateRideBody.kt
package com.lattis.data.entity.body.ride

import com.google.gson.annotations.SerializedName

class UpdateRideBody(@field:SerializedName("trip_id")
                     private val trip_id: Int,
                     @field:SerializedName("steps")
                     private val steps: Array<DoubleArray>)
